pkg/api/authz.go

```diff
@@ -18,11 +18,13 @@ import (
 )
 
 const (
-	// actions.
+	// method actions.
 	CREATE = "create"
```

Comment:
@peusebiu change these to golang camelcase. These are not special, so no reason to do it this way.

Reply:
also remove the nolint